internal/providers/system/system.go

```diff
@@ -30,12 +32,25 @@ import (
 )
 
 const (
-	baseFilename = "base.ign"
 	userFilename = "user.ign"
 )
 
-func FetchBaseConfig(logger *log.Logger) (types.Config, report.Report, error) {
-	return fetchConfig(logger, baseFilename)
+func FetchBaseConfig(logger *log.Logger, platformName string) (types.Config, report.Report, error) {
+	var fullBaseConfig types.Config
+	baseDConfig, baseDReport, err := fetchBaseConfig(logger, "base.d")
+	if err != nil {
+		logger.Warning("no config at base.d: %v", err)
```

Comment:
The callee is inconsistent about when it logs errors and when it doesn't, so this may or may not log a double message.  We should clean that up.  Also, the message here is incorrect; if there's _no_ config we'll get a `nil` error.

Reply:
So instead of returning a customized error using `logger.Warning()`, I will simply return the `err`.